src/fsharp/FSharp.Core/SR.fs

```diff
@@ -26,13 +26,11 @@ module internal SR =
     let inputMustBeNonNegative = "inputMustBeNonNegative"
     let inputMustBePositive = "inputMustBePositive"
     let enumerationPastIntMaxValue = "enumerationPastIntMaxValue"
-    let inputSequenceEmpty = "inputSequenceEmpty"
```

Comment:
Resource name strings are now split across or duplicated in 3 different modules - `SR`, `Resources`, and `ErrorStrings`... I'd ideally like us to find a way to keep things better organized.